### PR TITLE
Adding Minex to BTC coins

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by btchip** [comment]
It's not ok for Bitcoin based altcoins to use Bitcoin P2PKH / P2SH version while not providing replay protection through the fork id mechanism (ideally both shouldn't happen at all)

**Comment by ghost** [comment]
So are you saying I should remove --path "44'/0'" ?

Or are you saying that minex wont get ledger support because they didn't set up their blockchain correctly?

**Review by btchip — CHANGES_REQUESTED** [review]
Change to specific P2PKH/P2SH versions and add replay protection

File: Makefile [hunk 1]
```diff
@@ -132,6 +132,14 @@ else ifeq ($(COIN),hcash)
 DEFINES   += COIN_P2PKH_VERSION=40 COIN_P2SH_VERSION=100 COIN_FAMILY=2 COIN_COINID=\"HShare\" COIN_COINID_HEADER=\"HCASH\" COIN_COLOR_HDR=0x57448D COIN_COLOR_DB=0xABA2C6 COIN_COINID_NAME=\"HCash\" COIN_COINID_SHORT=\"HCASH\" COIN_KIND=COIN_KIND_HCASH COIN_FLAGS=FLAG_PEERCOIN_SUPPORT
 APPNAME ="HCash"
 APP_LOAD_PARAMS += --path $(APP_PATH)
+else ifeq ($(COIN),minex)
+# Minex mainnet
+DEFINES   += COIN_P2PKH_VERSION=0 COIN_P2SH_VERSION=5 COIN_FAMILY=1 COIN_COINID=\"Minex\" COIN_COINID_HEADER=\"MINEX\" COIN_COLOR_HDR=0xFCB653 COIN_COLOR_DB=0xFEDBA9 COIN_COINID_NAME=\"Bitcoin\" COIN_COINID_SHORT=\"BTC\" COIN_NATIVE_SEGWIT_PREFIX=\"bc\" COIN_KIND=COIN_KIND_BITCOIN COIN_FLAGS=FLAG_SEGWIT_CHANGE_SUPPORT
```
Comment (MinexRnD) [inline Makefile:143]: COIN_P2PKH_VERSION=75 COIN_COINID_NAME=\"Minexcoin\" COIN_COINID_SHORT=\"MNX\"

File: Makefile [hunk 2]
```diff
@@ -132,6 +132,14 @@ else ifeq ($(COIN),hcash)
...
+DEFINES_LIB=# we're not using the lib :)
+APPNAME ="Minex"
+APP_LOAD_PARAMS += --path "44'/182'" --path "44'/0'"
```
Comment (MinexRnD) [inline Makefile:146]: APP_LOAD_PARAMS += --path $(APP_PATH)

**Comment by ghost** [comment]
Thanks MinexRnD.

Bchip, I'm not sure how to resolve the merge conflicts. Looks like two different coins, and then a compiler flag.

Can you merge, or want me to try? Lemme know.

**Comment by MinexRnD** [comment]
Hello Coranos!
Try merge by yourself, when I will have free time help you and support.

2018-04-20 5:04 GMT+03:00 coranos <notifications@github.com>:

**Comment by ghost** [comment]
I think I got all the merges working. can yo commit the minex change into BTC?

**Comment by ghost** [comment]
@btchip  I think I got all the merges working. can yo commit the minex change into BTC?

**Pull request closed without merging**